packages/bpk-component-infinite-scroll/readme.md

```diff
@@ -0,0 +1,110 @@
+# bpk-component-infinite-scroll
+
+> Backpack infinite scroll HOC component.
+
+## Installation
+
+```sh
+npm isntall bpk-component-infinite-scroll --save-dev
+```
+
+## Usage
+```js
+import React from 'react';
+import PropTypes from 'prop-types';
+import BpkInfiniteScroll from 'bpk-component-infinite-scroll';
```

Comment:
If this is exporting a HOC then its usage shouldn't be advertised like this. It's not a component therefore it should not be capitalised.

I suggest:
```
import withInfiniteScroll from 'bpk-component-infinite-scroll';
// or 
import { withInfiniteScroll } from 'bpk-component-infinite-scroll';
```

I suggest the second example because usually the default export is a component ... not sure if we can actually export a component here - why should it be a HOC?

Reply:
It's a function that takes a `component` and returns another `component`. If we turned it into a normal component it would be an infinite list (?) with scrolling features. But I think that this way it acts as an infinite provider "listening for the scroll event", and lets you implement the specific list/table/whatever that will render that information.
Suggestions accepted, of course :D

Reply:
Indeed it's a function so lets go with `withInfiniteScroll`. I'll comment where necessary for the renames.

As for default export versus named export ... we should probably go for default export given that it's the only thing thats exported.

Reply:
```
import withInfiniteScroll from './src/withInfiniteScroll';

import onRequestElements from './src/defaultRequestElements';

export withInfiniteScroll { onRequestElements };```

Reply:
Yup! 

Actually whilst we are here... :P the naming of `onRequestElements` is a little off. Usually (in react land at least), the verb is at the end of the prop name i.e. `onElementsRequest` or rather `onElementsFetch`? Given that it's an infinite scroll and the elements can actually be anything I'd prefer the term "items". So ... how about `onItemsFetch`? 

cc @tiagoengel  what do you think?

Reply:
I like `onItemsFetch`, it's more descriptive 👍

Reply:
done